Chipotle/Program.cs

```diff
@@ -4,12 +4,23 @@ namespace Chipotle
 {
     class Program
     {
-        static void Main()
+        public static void Main()
         {
-            var port = 8000;
-            var server = new Chorizo.Chorizo(port, "TelNet");
+            var port = 5000;
+            var app = new Chorizo.Chorizo(port);
+
+            app.ProtocolConnectionHandler.Router().Get("/", (req, res) =>
```

Comment:
I don't adding routes should go in `Main`. It probably should live in a `Setup` or `Configuration` class